pom.xml

```diff
@@ -31,11 +37,6 @@
     <url>http://microprofile.io</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-
-        <checkstyle.version>3.0.0</checkstyle.version>
```

Comment:
Most of the lines in this pom.xml can be deleted.

Reply:
Most of the lines deleted